third_party/requirements.txt

```diff
@@ -2,6 +2,6 @@
 # Formatting the source code
 clang-format==15.0.7
 # Generating HTML documentation
-breathe==4.35.0
-sphinx==4.5.0
-sphinx_book_theme==0.3.3
+breathe>=4.35.0
+sphinx>=5.0.0
+sphinx_book_theme>=0.3.3
```

Comment:
Please, update the commit message with some details why do we need this change.

Reply:
I changed my PR because I wanted to keep the current layout